Sources/StreamChatUI/ChatMessageList/ChatMessageListView.swift

```diff
@@ -259,6 +262,28 @@ open class ChatMessageListView: UITableView, Customizable, ComponentsProvider {
         }
     }
 
+    private func getVisibleQuotedMessageUpdates(forUpdatedMessageAt indexPath: IndexPath) -> [IndexPath]? {
+        let visibleCells = getVisibleCells()
+        guard let visibleUpdatedMessage = visibleCells[indexPath]?.messageContentView?.content else {
```

Comment:
Please use safe index :)

Reply:
No need for safe index, its  actually a dictionary 😋